core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitManager.java

```diff
@@ -323,14 +310,8 @@ public void mutate(List<? extends Mutation> mutations)
   private <R> R executeTransaction(
       ThrowableFunction<DistributedTransaction, R, TransactionException> throwableFunction)
       throws CrudException, UnknownTransactionStatusException {
-    DistributedTransaction transaction;
-    try {
-      transaction = beginInternal();
-    } catch (TransactionNotFoundException e) {
-      throw new CrudConflictException(e.getMessage(), e, e.getTransactionId().orElse(null));
-    } catch (TransactionException e) {
-      throw new CrudException(e.getMessage(), e, e.getTransactionId().orElse(null));
-    }
+    String txId = UUID.randomUUID().toString();
+    DistributedTransaction transaction = begin(txId);
```

Comment:
```suggestion
    DistributedTransaction transaction = begin();
```

[minor] A bit simpler?

Reply:
Fixed in 41b597a0ce93ed0b3c9b9d3d3bd475eabde82c22. Thanks!